ratpack-codahale-metrics/src/test/groovy/ratpack/codahale/metrics/MetricsSpec.groovy

```diff
@@ -73,6 +74,7 @@ class MetricsSpec extends RatpackGroovyDslSpec {
         .jmx()
         .csv { it.reportDirectory(reportDirectory.root).reporterInterval(Duration.ofSeconds(1)) }
         .console { it.reporterInterval(Duration.ofSeconds(1)) }
+        .graphite { it.enable().sender(new Graphite("localhost", 2003)).reporterInterval(Duration.ofSeconds(1)) }
```

Comment:
We need to test that this is actually working.  I think we should be able to use a test ratpack app to receive and assert, like this https://github.com/ratpack/ratpack/blob/master/ratpack-core/src/test/groovy/ratpack/http/client/HttpProxySpec.groovy#L31